### PR TITLE
Rename shell variable MIRAGE_EXTRA_REPO to EXTRA_REPOS (we can add multiple repositories)

### PR DESCRIPTION
**Pull request by dinosaure**

### CODE REVIEW

File: lib/functoria/cli.ml [hunk 1]
```diff
@@ -89,9 +89,9 @@ let extra_repo doc_section =
     let pp ppf (name, repository) = Fmt.pf ppf "%s:%s" name repository in
     Arg.conv (parser, pp)
   in
-  let env = Arg.env_var "MIRAGE_EXTRA_REPO" in
+  let env = Arg.env_var "MIRAGE_EXTRA_REPOS" in
   let doc =
-    Arg.info ~docs:doc_section ~docv:"URL" ~env
+    Arg.info ~docs:doc_section ~docv:"(<name>:<url>)+" ~env
```
Comment (TheLortex) [inline lib/functoria/cli.ml:94]: `(<name>:<url>)+` doesn't really make it clear that the repos are comma-separated

**Review by TheLortex — CHANGES_REQUESTED** [review]
I agree that it should be plural, but in that case the command line argument name should also be changed (`--extra-repo` => `--extra-repos`)

File: lib/functoria/cli.ml (hunk 1 above)
Comment (dinosaure) [reply lib/functoria/cli.ml:94]: hmmhm, do you prefer: `<name>:<url>(','<name>:<url>)*`?

Comment (TheLortex) [reply lib/functoria/cli.ml:94]: Maybe `NAME1:URL1,NAME2:URL2,...` ?

**Review by TheLortex — APPROVED** [review]

**Pull request merged**